pkg/apiserver/slowquery/queries.go

```
@@ -166,8 +196,11 @@ func QuerySlowLogList(db *gorm.DB, req *GetListRequest) ([]Base, error) {
 		tx = tx.Where("DB IN (?)", req.DB)
 	}
 
-	order := req.OrderBy
-	if isValidColumnName(order) {
+	order, err := getRefColumn(req.OrderBy)
+	if err != nil {
+		log.Error("req.OrderBy is invalid", zap.String("orderBy", req.OrderBy), zap.Error(err))
```

In this case, should we just return error instead of letting it continue?